src/components/Carousel/Carousel.tsx

```diff
@@ -109,6 +113,10 @@ const Carousel: React.FC<CarouselProps> = ({
   indicatorsActiveColor,
   indicatorsInactiveColor,
   isIndicatorsShadow = true,
+  selectable = false,
+  pauseOnHover = false,
+  onClickItem,
```

Comment:
For the same reason, `onClickItem` can also be removed here.

Reply:
idem